packages/vaadin-icon/src/vaadin-icon.d.ts

```diff
@@ -4,11 +4,53 @@ import { IconSvgLiteral } from './vaadin-icon-svg.js';
 
 /**
  * `<vaadin-icon>` is a Web Component for creating SVG icons.
+ *
+ * ### Icon property
+ *
+ * The `<vaadin-icon>` component is designed to be used as a drop-in replacement for `<iron-icon>`.
+ * For example, you can use it with `vaadin-icons` like this:
```

Comment:
Would leave out mentions of the Polymer's `<iron-icon>` altogether. The feature set of the two elements aren't equal so it probably doesn't work as a drop-in replacement.

Reply:
Good point. I think we can remove this part indeed.